packages/amplify-category-auth/tests/commands/enable.test.js

```diff
@@ -0,0 +1,62 @@
+const add = require('../../commands/auth/enable');
+
+describe('auth enable: ', () => {
+  const mockExecuteProviderUtils = jest.fn();
+  const mockGetProjectDetails = jest.fn();
+  const mockSelectionPrompt = jest.fn(() => Promise.resolve({}));
+  const mockProjectPath = '/User/someone/Documents/Project/amplify-test';
+  const mockContext = {
+    amplify: {
+      executeProviderUtils: mockExecuteProviderUtils,
+      getProjectDetails: mockGetProjectDetails,
+      serviceSelectionPrompt: mockSelectionPrompt,
+    },
+    print: {
+      warning: jest.fn(),
+    },
+  };
+
+
+  afterEach(() => {
+    mockGetProjectDetails.mockReturnValue({});
+  });
+
+  it('enable run method should exist', () => {
+    expect(add.run).toBeDefined();
+  });
+
+  describe('case: auth already enabled', () => {
+    beforeEach(() => {
+      mockGetProjectDetails.mockReturnValue({
+        projectConfig: {
+          projectPath: mockProjectPath,
+        },
+        amplifyMeta: {
+          auth: {
+            foo: 'bar',
+          },
+        },
+      });
+    });
+    it('enable method should detect existing auth metadata', () => {
+      add.run(mockContext);
+      expect(mockContext.print.warning).toBeCalledWith('Auth has already been added to this project. To update run amplify update auth');
```

Comment:
Can we just checked if the method is called instead of `toBeCalledWith` as the message is not really relevant to the test

Reply:
done.